templates/21-loggly-application.conf

```diff
@@ -0,0 +1,64 @@
+$ModLoad imfile
+$InputFilePollInterval 10
+$PrivDropToGroup adm
+
+#Add a tag for nginx events
+$template LogglyFormatNginx,"<%pri%>%protocol-version% %timestamp:::date-rfc3339% %HOSTNAME% %app-name% %procid% %msgid% [{{ loggly.token }}@41058 tag=\"nginx\"] %msg%\n"
+
+$WorkDirectory /var/spool/rsyslog # where to place spool files
+$ActionQueueFileName fwdRule1     # unique name prefix for spool files
+$ActionQueueMaxDiskSpace 1g       # 1gb space limit (use as much as possible)
+$ActionQueueSaveOnShutdown on     # save messages to disk on shutdown
+$ActionQueueType LinkedList       # run asynchronously
+$ActionResumeRetryCount -1        # infinite retries if host is down
+
+#RsyslogGnuTLS
+$DefaultNetstreamDriverCAFile {{ loggly.certificate_dest }}{{ loggly.certificate_file }}
+$ActionSendStreamDriver gtls
+$ActionSendStreamDriverMode 1
+$ActionSendStreamDriverAuthMode x509/name
+$ActionSendStreamDriverPermittedPeer *.loggly.com
+
+# Nginx access file:
+{% for filename in loggly.nginx.access_logs  %}
+$InputFileName {{ filename }}
```

Comment:
it is - i debated because I wanted the formatting to be correct on the Ansible target server itself - but either should work.